librad/src/git/repo.rs

```diff
@@ -98,7 +98,10 @@ impl<'a> Repo<'a, WithSigner> {
     /// Fetch new refs and objects for this repo from [`PeerId`]
     pub fn fetch(&self, from: &PeerId) -> Result<(), Error> {
         self.storage
-            .fetch_repo(&self.urn, from)
+            .fetch_repo(RadUrl {
```

Comment:
Not really, `RadUrl` has one additional piece of information over `RadUrn`, so it's not an isomorphism, so `From`, `Into`, et. al. don't make sense. There is `RadUrn::into_rad_url`, but I came to dislike this a bit over just the constructors, because of named fields.